op-e2e/setup.go

```diff
@@ -214,12 +214,12 @@ type System struct {
 	Mocknet           mocknet.Mocknet
 }
 
-func (sys *System) Close() {
+func (sys *System) Close(ctx context.Context) {
 	if sys.L2OutputSubmitter != nil {
 		sys.L2OutputSubmitter.Stop()
 	}
 	if sys.BatchSubmitter != nil {
```

Comment:
```suggestion
func (sys *System) Close() {
	if sys.L2OutputSubmitter != nil {
		sys.L2OutputSubmitter.Stop()
	}
	
	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
	defer cancel()
	if sys.BatchSubmitter != nil {
```

Instead of adding a `ctx` parameter to `Close`, we could also consider to just always close the batcher with some default context (like you do in the `TestMissingBatchE2E` test). You pass `context.Background()` to all other tests anyways. 

But I'm not that familiar with the e2e test, would like to hear @trianglesphere opinion here.

Reply:
This definitely sounds reasonable enough to me for the purpose of e2e tests, but happy to change course if there's a good reason to not take this approach.